pkgs/by-name/fi/finamp/package.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, flutter
+, mpv
```

Comment:
```suggestion
, mpv-unwrapped
```

Not so sure, but usually we don't use wrapped ones if we need the library

Reply:
Fixed